packages/ui/components/combobox/src/LionCombobox.js

```diff
@@ -201,8 +202,24 @@ export class LionCombobox extends LocalizeMixin(OverlayMixin(CustomChoiceGroupMi
 
   reset() {
     super.reset();
-    // @ts-ignore _initialModelValue comes from ListboxMixin
-    this.value = this._initialModelValue;
+    if (!this.multipleChoice) {
+      // @ts-ignore _initialModelValue comes from ListboxMixin
+      this.value = this._initialModelValue;
+    }
+    this.resetListboxOptions();
+  }
+
+  resetListboxOptions() {
```

Comment:
Can this be a private method?

Reply:
Protected is possible, we need to override it internally.